extension-config.json

```diff
@@ -31,6 +31,17 @@
         "key": "value"
       }
     },
+    "registerAccountUrl": {
```

Comment:
This is necessary, because the /register route cant be changed to anything else at this point and it'll probably take quite some time until it will be changed. This is here to lift a blocker for us and can be useful to externals.